workbench-core/accounts/src/models/account.ts

```diff
@@ -5,7 +5,7 @@
 
 import { HostingAccountStatus } from '../constants/hostingAccountStatus';
 
-interface Account {
+export default interface Account {
```

Comment:
Btw, instead of using an interface. Could we just use a `type`? 
```
export type Account =  z.infer<typeof AccountParser>;
```

Reply:
This way we don't have to define an account twice. Once in the parser and the second time in the interface.

Reply:
I can but then everything is defined in terms of Zod types, which I'm not sure we want to do. The repeated declarations is a little annoying but it saves us readability 🤷 . Thoughts?

<img width="787" alt="Screen Shot 2022-11-08 at 12 53 57 PM" src="https://user-images.githubusercontent.com/112508158/200650969-e951c333-4006-4253-9c74-61531ebc3743.png">

Reply:
When you actually instantiate the type, i believe that get cleaned up and you just see `string` etc, but not when you're defining the `type`

Reply:
Actually it doesn't 😢. I'd suggest keeping both declarations for now so we don't have to use Zod types everywhere. Its an unfortunate trade off but that's what happens when static type analysis is built in as an afterthought at compile time. 
<img width="858" alt="Screen Shot 2022-11-08 at 12 56 43 PM" src="https://user-images.githubusercontent.com/112508158/200651502-3eb569b3-65fe-4266-beb6-d52d99abd481.png">

Reply:
Ah I see. In that case I'm ok with defining Account twice.